docs/hoon/library/2k.md

```diff
@@ -8,38 +8,41 @@ navhome: /docs/
 
 Queue operations
 
-Container arm for queue operation arms. The contained arms inherit its
-sample `++qeu` `a`.
+Container arm for queue operation arms. The contained arms inherit `a`, its
+sample queue.
 
-Accepts
--------
+#### Accepts
 
-`a` is a queue, ++qeu.
+`a` is a tree.
 
-Source
-------
+#### Source
 
-    ++  to                                                  ::  queue engine
-      |/  a/(qeu)
+    ++  to                                                 
+      |_  a=(tree)
 
+#### Discussion
+
+`tree` is the generic tree-shape mold. Maps, sets, and queues all share that
+shape, but they put their elements in different places and have different
```

Comment:
The places where the elements can go are the same; they're just ordered differently. I'd say something more like:
`but they store their elements in different orders and have different interfaces`.